packages/compile-solidity-tests/test/compileWithPragmaAnalysis.ts

```diff
@@ -129,8 +133,11 @@ describe("compileWithPragmaAnalysis", function () {
         options: config,
         paths: [path.join(sourceDirectory, "withImports", "C.sol")]
       });
+      if (!compilations) {
+        assert.fail("Compilations were not created.");
+      }
       assert.equal(compilations.length, 1);
-      assert(compilations[0].compiler.version.startsWith("0.6.12"));
+      assert(compilations[0].compiler.version!.startsWith("0.6.12"));
```

Comment:
What is `!.`?  I've never seen that before, I can't find any documentation on it...

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-0.html#non-null-assertion-operator

Reply:
Yeah, that just tells TS "hey, I know what I'm doing and this will be defined". It's just so I don't have to do something like
```
if (compilations[0] && compilations[0].compiler && compilations[0].compiler.version...
```

Reply:
Oh, OK.  Thanks!